infra/wca_on_rails/production/rails.tf

```diff
@@ -206,14 +206,14 @@ resource "aws_ecs_task_definition" "this" {
   task_role_arn      = aws_iam_role.task_role.arn
 
   cpu = "8192"
-  memory = "30000"
+  memory = "31634"
```

Comment:
This RAM value was also set wrongly (we were leaving 1.6GB of RAM on the table, not that that was ever an issue...)